measures/ResStockArguments/measure.rb

```diff
@@ -466,6 +466,64 @@ def run(model, runner, user_arguments)
       args['heat_pump_charge_defect_ratio'] = args['heat_pump_frac_manufacturer_charge'].get - 1.0
     end
 
+    # Infiltration adjustment for SFA/MF units
+    if [HPXML::ResidentialTypeApartment, HPXML::ResidentialTypeSFA].include? args['geometry_unit_type']
+      n_units = Float(args['geometry_building_num_units'])
+      n_floors = Float(args['geometry_num_floors_above_grade'])
+      aspect_ratio = Float(args['geometry_aspect_ratio'])
+      horiz_location = args['geometry_horizontal_location'].to_s
+      corridor_position = args['geometry_corridor_position'].to_s
+
+      if args['geometry_unit_type'] == HPXML::ResidentialTypeApartment
+        n_units_per_floor = n_units / n_floors
+        if (n_units_per_floor >= 4) && (corridor_position != 'Single Exterior (Front)') # assume double-loaded corridor
+          has_rear_units = true
+        elsif (n_units_per_floor == 2) && (horiz_location == 'None') # double-loaded corridor for 2 units/story
+          has_rear_units = true
+        else
+          has_rear_units = false
+        end
+      elsif args['geometry_unit_type'] == HPXML::ResidentialTypeSFA
+        n_floors = 1.0
+        n_units_per_floor = n_units
+        has_rear_units = false
+      end
+
+      # Calculate exposed wall area ratio for the unit (unit exposed wall area
+      # divided by average unit exposed wall area)
+      if (n_units_per_floor == 1) || (n_units_per_floor == 2) || ((n_units_per_floor == 4) && has_rear_units) # No middle unit(s)
```

Comment:
I think this logic should be updated to `if (n_units_per_floor <=2) || ((n_units_per_floor == 4) && has_rear_units)`, as non-integer units/floor are now valid as of this PR: https://github.com/NREL/resstock/pull/558. 

Same thing would need fixed on `sync-tsvs`/`develop`, I can do that.

Reply:
Hmm, I am looking through that PR and not seeing where non-integer values are allowed. Can you more directly show me?

Reply:
https://github.com/NREL/resstock/pull/558/files?file-filters%5B%5D=.tsv#diff-26529a8eddbe8c9d6be409cf357bead9406fbbc929779ee3e2107371f859d3be

`Geometry Building Number Units MF` is no longer dependent on `Geometry Stories`. I suppose non-integer values were always valid, but now we actually get them from the sampling.

Reply:
Ah thanks, I was looking for some kind of integer -> double change. I'll make your proposed change.